source/win-main/PopoverExport.vue

```diff
@@ -65,8 +73,8 @@ ipcRenderer.invoke('assets-provider', { command: 'list-export-profiles' } as Ass
     // on the value of "format".
     profileMetadata.value = defaults
     // Get either the last used exporter OR the first element available
-    const lastProfile = configStore.config.export.singleFileLastExporter
-    if (lastProfile in availableFormats.value) {
+    const lastProfile = selectedProfiles.value.get(props.filePath)
```

Comment:
Here, I want you to fetch the lastProfile using the selectedProfiles props.filePath, BUT if it is not found in the recently selected profiles, fall back to the global setting.